src/main/java/commands/FindCmd.java

```diff
@@ -0,0 +1,47 @@
+package commands;
+
+import java.util.List;
+
+import drivers.Storage;
+import drivers.TaskList;
+import drivers.Ui;
+import exceptions.TumuException;
+import tasks.Task;
+
+/**
+ * Class to be executed when a find command is issued
+ * by the user.
+ *
+ * Finds all entries that contain the keyword inside the task description.
+ */
+public class FindCmd extends Command {
```

Comment:
I would spell out the full class, `FindCommand`. Try to avoid unnecessary acronyms when you can.